challenges/challenges_app/urls.py

```diff
@@ -9,4 +9,7 @@
     path("signup", views.signup_view, name="signup"),
     path("account", views.account_view, name="account"),
     path("problems/<str:title>", views.problem_view, name="problems"),
+    path("search/", views.search, name="search-problems"),
+    path("submit/new-problem", views.add_problem, name="add-problem"),
```

Comment:
same as previous comment

Reply:
Agree. Submits should be done via Github pull request on the problems repo.